var/spack/repos/builtin/packages/py-seaborn/package.py

```diff
@@ -36,6 +36,7 @@ class PySeaborn(PythonPackage):
     homepage = "http://seaborn.pydata.org/"
     url      = "https://pypi.io/packages/source/s/seaborn/seaborn-0.7.1.tar.gz"
 
+    version('0.9.0', '76c83f794ca320fb6b23a7c6192d5e185a5fcf4758966a0c0a54baee46d41e2f')
```

Comment:
should this be sha256=... ?

Reply:
Works like this, too. I've seen other packages where the hashing algo isn't specified.